src/card/frame_effect.rs

```diff
@@ -68,6 +68,8 @@ pub enum FrameEffect {
     Textless,
     /// Is a story spotlight card
     StorySpotlight,
+    /// Undocumented
+    Gilded,
```

Comment:
Is this undocumented in scryfall's webiste?

Reply:
at least i couldnt find anything about it in their api docs

Reply:
thinking about it, it might be [this](https://secretlair.wizards.com/eu/de/product/724663/showcase-streets-of-new-capenna-gilded-foil-edition)